specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -810,6 +816,10 @@
           "items": {
             "$ref": "#/definitions/HealthcareRelation"
           }
+        },
+        "fhirBundle": {
+          "type": "object",
```

Comment:
I think we need to add `additionalProperties` to the definition to get the expected generation:

```
"additionalProperties": {
  "type": "string"
}
```

@johanste to confirm.

Reply:
(Speaking from an openapi/swagger perspective) the current definition (which includes additionalProperties: string) only allows for string values. If you omit the "additionalProperties" and don't have a "properties" attribute, then it could be string-to-any-kind-of-values (e.g. integers, nested objects etc.). So it depends on the domain of valid values.

From an autorest perspective, I am not 100% sure what code we generate, but if there is an issue here, we should resolve it on the codeine side.

Reply:
@thediris will FHIR contain non-string values?

Reply:
@kristapratico deep inside the object, I see arrays and numbers: https://www.jsonblob.com/966114210698510336

I think it should be represented as an obaque JSON object but not sure how to convey that in swagger terms.